src/pages/AddressInfoPage/AddressTransactionRow.tsx

```diff
@@ -111,72 +113,111 @@ const AddressTransactionRow: FC<AddressTransactionRowProps> = ({ transaction: t,
 
         <Badge
           type="neutralHighlight"
-          content={infoType === 'move' ? 'Moved' : infoType === 'out' ? 'To' : infoType === 'swap' ? 'Swap' : 'From'}
+          content={
+            infoType === 'move'
+              ? 'Moved'
+              : infoType === 'out'
+              ? 'To'
+              : infoType === 'swap'
+              ? 'Swap'
+              : infoType === 'pending'
+              ? 'Pending'
+              : 'From'
+          }
```

Comment:
nit: How about moving this into a variable instead?

```js
const badgeText = {
  move: 'Moved',
  out: 'To',
  swap: 'Swap',
  pending: 'Pending',
  in: 'From'
}[infoType]
```

even better, it would perfectly fit in `src/hooks/useTransactionUI.ts`.

Reply:
very good idea!